### DISCUSSION TITLE
要望：アイコンを大きくしてほしい

### DISCUSSION THREAD

**Discussion by Makohan** [increments/qiita-discussions#169] [要望・アイデア・不具合]
## 問題
記事一覧や詳細記事に掲載されているユーザーのアイコンが小さくて視認性が悪く感じます。

## 期待される効果
- ユーザーアイコンの視認性の向上
- ユーザーアイコンはユーザーのアバターなので、大きくすることで個性を尊重する意味も込められる

**Comment by kskwtnk** [comment]
@Makohan 
ご意見ありがとうございます！

こちら、実は現在着手している[QiitaのUIアップデート](https://blog.qiita.com/ui-update-plan-for-2022/)の一環で進めております！
まずは近々ベータ版としてリリース予定です！

**Reply by Makohan** [reply]
@xrxoxcxox 
ご回答ありがとうございます！

既に進んでいるんですね。ベータ版のリリース楽しみにしております！

**Reply by kyntk** [reply]
@Makohan 記事カードのUI変更をベータ版として公開しました！
ぜひご利用ください！ 🙏🏽 
https://qiita.com/release-notes#2022-03-22

**Reply by Makohan** [reply]
@kyntk 
確認しました！
アイコンが大きくなってました、ありがとうございます！

記事内のアイコンが小さいままなのが気になりましたが、
そちらも今後、ご検討いただけると幸いです。
